ginga/rv/Control.py

```diff
@@ -608,6 +608,14 @@ def get_fileinfo(self, filespec, dldir=None):
         # Get information about this file/URL
         info = iohelper.get_fileinfo(filespec, cache_dir=dldir)
 
+        if isinstance(info, list):
```

Comment:
I am wondering if we should just make `get_fileinfo()` *always* return a list.  If there is only one file selected, it would return a list of one bunch.  I think this makes more sense than having to check the results everywhere that it is used.